src/introduction/introduction.component.js

```diff
@@ -29,49 +34,54 @@ export default function Introduction(props) {
           applications.
         </section>
         <section>
-          The playground isn't the only way to use single-spa, but is one of the
-          easier ways. It will set you up with single-spa,{" "}
-          <a href="https://github.com/systemjs/systemjs">systemjs@6</a>, and{" "}
-          <a href="https://github.com/WICG/import-maps">import maps</a>. This
-          lets you do the following:
+          The playground will guide you to build a successful sing-e-spa
+          application, highlighting important steps, and giving you tools to
+          help you understand what went wrong. Mind that this isn't the only way
+          to use single-spa, but by better understanding the concepts you'll be
+          able to apply it in any way you like. Here's what we'll do:
           <ol>
-            <li>Have microfrontends that are in separate git repos.</li>
             <li>
-              Use different frameworks in the microfrontends, if you wish.
+              Have micro-frontends that are in separate git repos. We will be
```

Comment:
Better to have consistent spelling of this I think. So to match other places:

```suggestion
              Have microfrontends that are in separate git repos. We will be
```